docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,321 @@ redirect_from:
+
+Under the **Enable Events** switch in the **Request URL** box, go ahead and paste in your URL. As long as your Bolt app is still running, your URL should become verified.
+
+After your request URL is verified, scroll down to **Subscribe to Bot Events**. There are four events related to messages:
+- `message.channels` listens for messages in public channels that your app is added to
+- `message.groups` listens for messages in private channels that your app is added to
+- `message.im` listens for messages in the App Home/DM space
+- `message.mpim` listens for messages in multi-person DMs that your app is added to
+
+If you want your bot to listen to messages from everywhere it is added to, choose all four message events. After you’ve selected the events you want your bot to listen to, click the green **Save Changes** button.
+
+---
+
+### Listening and responding to a message
+Your app is now ready for some logic. Let's start by using the `message()` method to attach a listener for messages.
+
+The following example listens and responds to all messages in channels/DMs where your app has been added that contain the word "hello":
+
+```python
+import os
+from slack_bolt import App
+
+# Initializes your app with your bot token and signing secret
+app = App(
+    token=os.environ.get("SLACK_BOT_TOKEN"),
+    signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+# Listens to incoming messages that contain "hello"
+@app.message("hello")
+def message_hello(message, say):
+    # say() sends a message to the channel where the event was triggered
+    say(f"Hey there <@{message['user']}>!")
+
+# Start your app
+if __name__ == "__main__":
+    app.start(port=int(os.environ.get("PORT", 3000)))
+```
+
+If you restart your app, so long as your bot user has been added to the channel/DM, when you send any message that contains "hello", it will respond.
+
+This is a basic example, but it gives you a place to start customizing your app based on your own goals. Let's try something a little more interactive by sending a button rather than plain text.
+
+---
+
+### Sending and responding to actions
+
+To use features like buttons, select menus, datepickers, dialogs, and shortcuts, you’ll need to enable interactivity. Similar to events, you'll need to specify a URL for Slack to send the action (such as *user clicked a button*).
+
+Back on your app configuration page, click on **Interactivity & Shortcuts** on the left side. You'll see that there's another **Request URL** box.
+
+By default, Bolt is configured to use the same endpoint for interactive components that it uses for events, so use the same request URL as above (in the example, it was `https://8e8ec2d7.ngrok.io/slack/events`). Press the **Save Changes** button in the lower right hand corner, and that's it. Your app is set up for interactivity!
```

Comment:
```suggestion
By default, Bolt is configured to use the same endpoint for interactive components that it uses for events, so use the same request URL as above (in the example, it was `https://8e8ec2d7.ngrok.io/slack/events`). Press the **Save Changes** button in the lower right hand corner, and that's it. Your app is set up to handle interactivity!
```

Reply:
Thanks for the suggestion, @shaydewael! I've also updated this in `bolt-js` to keep everything aligned.